langDir/en.json

```diff
@@ -520,7 +520,9 @@
     "caller": "Maker",
     "price": "Price",
     "expiration": "Expiration",
-    "action": "Action"
+    "action": "Action",
+    "collection": "Collection",
+    "nftName": "Name"
```

Comment:
Hey! looks like this 2 value already exist in en.json file, no need to duplicate them